documentation/new-version-quickstart.md

```diff
@@ -0,0 +1,692 @@
+
```

Comment:
this is the Track 2 quickstart and will eventually be moved into Microsoft Docs. Providing it here for now so customer can access it